src/lib/device_tree/tests/integration_test.rs

```diff
@@ -37,7 +37,12 @@ fn dts_to_dtb(dts: &str) -> std::vec::Vec<u8> {
 
 fn read_all(d: &impl Driver) -> std::vec::Vec<u8> {
     let mut data = [0; 500];
-    let size = d.pread(&mut data, 0).unwrap();
+    let size = match d.pread(&mut data, 0) {
+        Ok(x) => Ok(x),
+        Err("EOF") => Ok(0),
+        y => y,
```

Comment:
I had to think about this a while before understanding that the `y => y` match arm was to catch any `Err` strings other than `"EOF"`. Normally I think of Ok + Err cases being the complete set, so this stood out to me. How would you feel about making this `Err(other) => Err(other)` for human clarity?

Reply:
I agree it is much more readable, thanks!